old mode 100644
new mode 100755
index cd75307..314a90b
--- a/arch/x86/entry/syscalls/syscall_64.tbl
+++ b/../linux-4.4.1_old/arch/x86/entry/syscalls/syscall_64.tbl
@@ -371,5 +371,3 @@
 543    x32     io_setup                compat_sys_io_setup
 544    x32     io_submit               compat_sys_io_submit
 545    x32     execveat                stub_x32_execveat
-546    64      mycounter_1             sys_mycounter_1
-547    64      mycounter_2             sys_mycounter_2

